### PR TITLE
Script directory

### PR DESCRIPTION
**Pull request by gregcorbett**
- Add a script directory and move some files from conf/ to scripts/
- Alter dockerfile and "setup from source" instructions accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,23 +33,61 @@ Experimental REST API for APEL
     ```
     
 3. Install requirements.txt
+    ```
+    pip install -r requirements.txt
```
Comment (tofu-rocketry) [inline README.md]: `MySQL-python` is in `requirements.txt` and the `yum install` arguments. This correct?

File: README.md [hunk 2]
```diff
@@ -33,23 +33,61 @@ Experimental REST API for APEL
...
+    ```
 
-9. Copy `conf/apel_rest_api.conf` to `/etc/httpd/conf.d/apel_rest_api.conf`
+12. Run `python manage.py collectstatic`
 
-10. Run `python manage.py collectstatic`
+13. Start Apache with `service httpd start`
 
-11. Start Apache with `service httpd start`
+14. Start the loader with `service apeldbloader-cloud start`
```
Comment (tofu-rocketry) [inline README.md]: When does the init script get installed for this to be possible?

Comment (gregcorbett) [inline README.md]: Step 9: `cp /var/www/html/scripts/apeldbloader-cloud /etc/init.d/apeldbloader-cloud`

Comment (tofu-rocketry) [inline README.md]: Ah, ok, just didn't spot it.

File: README.md (hunk 1 above)
Comment (gregcorbett) [inline README.md]: Will fix on the release branch

**Comment by tofu-rocketry** [comment]
This is fine to merge now.

**Pull request merged**